xcodeproj/internal/xcodeprojinfo.bzl

```diff
@@ -285,23 +282,14 @@ def _skip_target(
             return target.label
         if target_skip_type != skip_type.apple_test_bundle:
             return target.label
-        if _APPLE_INTERNAL_TEST_BUNDLE_SUFFIX not in info.xcode_target.label.name:
-            warn("""\
-Couldn't find '{suffix}' suffix in '{label}' with skip_type={type}, \
-replacement labels might not work as expected.
-
-Please, file a bug report here: \
-https://github.com/MobileNativeFoundation/rules_xcodeproj/issues/new?template=bug.md
-""".format(
-                suffix = _APPLE_INTERNAL_TEST_BUNDLE_SUFFIX,
-                label = info.xcode_target.label,
-                type = target_skip_type,
-            ))
-            return target.label
+
+        # As of https://github.com/bazelbuild/rules_apple/pull/1948 `bundle_name` can be used to
+        # name the bundle instead of the target name. Because of that we use `ctx.rule.attr.generator_name` here
+        # to ensure this is always a real target label.
```

Comment:
Can you wrap this to 80 char?

Reply:
Will do one sec

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/2248/commits/70a3bf92a12288cef2823d14eb62c5cdbd6ab047